sentry/templates/configmap-sentry.yaml

```diff
@@ -55,10 +55,10 @@ data:
     ##########
     # Google #
     ##########
-    {{- if .Values.google.clientId }}
-    auth-google.client-id: {{ .Values.google.clientId | quote }}
-    auth-google.client-secret: {{ .Values.google.clientSecret | quote }}
-    {{ end }}
+    {{- if and (.Values.google.clientId) (.Values.google.clientSecret) (not .Values.google.existingSecret) }}
```

Comment:
I think you can make it `{{- if and (.Values.google.clientId) (.Values.google.clientSecret) }}` that's enough.

We should pass either of the possibilities

Reply:
It's just additional step to protect against case where user define _.Values.google.clientId_ and _.Values.google.existingSecret_.